test/prober/configuration.go

```diff
@@ -0,0 +1,153 @@
+/*
+ * Copyright 2020 The Knative Authors
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package prober
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"knative.dev/eventing/pkg/apis/eventing/v1alpha1"
+	"knative.dev/eventing/test/lib"
+	duckv1 "knative.dev/pkg/apis/duck/v1"
+	"path"
+	"runtime"
+	"text/template"
+)
+
+const (
+	configName       = "wathola-config"
+	configMountPoint = "/.config/wathola"
+	configFilename   = "config.toml"
+	watholaEventNs   = "com.github.cardil.wathola"
+	healthEndpoint   = "/healthz"
+)
+
+var (
+	eventTypes = []string{"step", "finished"}
+)
+
+func (p *prober) deployConfiguration() {
+	p.annotateNamespace()
+	p.deploySecret()
```

Comment:
Why do we need a secret here?

Reply:
I should have used a config map instead of secret here. There isn't anything secret in that configuration file. I'll rewrite that to config map.